omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -273,9 +274,8 @@ protected PageableResult doSearch(RequestContext context) throws ResponseExcepti
 			else {
 				return new NeedsPaging<Order>(orders, context);
 			}
-		}
-		
-		return new EmptySearchResult();
+		} else throw new InvalidSearchException("Please specify patient parameter with valid patientUuid");
```

Comment:
Can you format this like the rest of the code base?

Reply:
Yes, sure